aerogear/mobile-security-service
# feat(ui): Get username from the backend and populate - else logout (A…

…EROGEAR-9032)

## Notes
Creating as a draft pr as it the backend work captured in [AEROGEAR-9039](https://issues.jboss.org/browse/AEROGEAR-9039). **Please do not merge this pr until that work is completed as it blocks the ui from loading**
 

## Motivation
JIRA: https://issues.jboss.org/browse/AEROGEAR-9032 

## What
Use the user data if it's returned
On getting the user data is failed, then the user should be redirected to the oauth sign in screen

## Verification Steps
1. Build the docker image locally
`docker build -t docker.io/aerogear/mobile-security-service:AEROGEAR-9032-verification .`

2. Push Image to dockerhub
`docker push docker.io/aerogear/mobile-security-service:AEROGEAR-9032-verification`

3. Deploy the image to the project in the community cluster
3a. Contact me for link to project
3b. Update the deployment config `image` to the new docker image
3c. Wait for the deployment to finish and the pod to come up.

4. Navigate to the mss route

5. You should be presented with the OAuth login screen. 

6. Log in

EDIT: The following step is no longer required as that will be implemented seperately.
7. As the backend logic for the user functionality has not yet been implemented the user will be redirected to the the log in screen.

## Checklist:


## Progress

- [x] UI implementation
- [x] Backend implementation
